config/routes.rb

```diff
@@ -2,5 +2,8 @@
   root 'homes#index'
   devise_for :users
 
-  resources :bucket_lists, only: [:index, :new, :create]
+  resources :bucket_lists, only: [:index, :new, :create, :edit]
+
+  patch '/bucket_lists/:id/edit', to: 'bucket_lists#update', as: 'bucket_list'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.